nodejs/nodejs.json

```
@@ -0,0 +1,40 @@
+{
+  "type": "nodejs",
+  "display": "Node JS",
+  "run": {
+    "stop": "stop",
+    "pre": [],
+    "post": [],
+    "arguments": [
+      "--max-old-space-size=${memory}",
+      "${start}"
+    ],
+    "program": "node"
+  },
+  "environment": {
+    "type": "standard"
+  },
+  "supportedEnvironments": [
+    {
+      "type": "standard"
+    }
+  ],
+  "data": {
+    "memory": {
+      "value": "1024",
+      "type": "integer",
+      "required": true,
+      "desc": "How much memory in MB to allocate to the Node js",
+      "display": "Memory (MB)",
+      "internal": false
+    },
+    "start": {
+      "value": "start.js",
```

This file is usually called `index.js`.
```suggestion
      "value": "index.js",
```